inf-clojure.el

```diff
@@ -702,6 +746,35 @@ is present it will be used instead of the current file."
     (when switch-to-repl
       (inf-clojure-switch-to-repl t))))
 
+(defcustom inf-clojure-reload-default t
+  "Controls whether `inf-clojure-reload' should reload or reload-all by default."
+  :type '(choice (const :tag "Reload" t)
+                 (const :tag "Reload All" nil))
+  :safe #'booleanp
+  :package-version '(inf-clojure . "2.2.0"))
+
+(defun inf-clojure-reload (arg)
+  "Send a query to the inferior Clojure for reloading the namespace.
+See variable `inf-clojure-reload-form' and
+`inf-clojure-reload-all-form'.
+
+The prefix argument ARG can change the behavior of the command:
+
+  - C-u M-x `inf-clojure-reload': prompts for a namespace name.
+  - M-- M-x `inf-clojure-reload': inverts the default reload command
+  - M-- C-u M-x `inf-clojure-reload': inverts default AND prompts for a namespace name."
```

Comment:
There is probably better syntax for this...

Reply:
http://ergoemacs.org/emacs/inline_doc.html

Anyways, this will do for now.

Reply:
Btw, having 3 ways to invoke something becomes pretty confusing very fast - normally people do just single and double prefix arguments. I think the default inversion is a bit too much. I'd just drop this and the config and do two commands probably.

Reply:
It's just that they are so similar that you usually think of it as same. That is the reason why the same key binding is used.

The inversion of the default can be indeed dropped and we can just always default for `reload`. With `M--` `reload-all`.
I have used this for a week now and found it good enough.